### DISCUSSION TITLE
Create 3 OR

### DISCUSSION THREAD

**Discussion by iansexton1** [iRobotEducation/create3_docs#120] [Questions] [answered]
In regards to the proximity sensors, when I try to use the robot.set_lights_rgb(r, g, b) command I set  r = 255 * sensors[2] / 4095 for example the red light won't appear until we get values on column 3 on the feedback array (see attached image) 

.  In addition, when we try to assign an rgb value for sensor[6] the code seems to crash. Is sensor 6 unable to be used for the command robot.set_lights_rgb(r, g, b)?

![IR Optical Sensor feedback](https://user-images.githubusercontent.com/109889851/180633869-3a19950d-7588-452d-8bf0-f0e0c7129731.jpg)

**Comment by shamlian** [comment, marked as answer]
This question is somewhat unclear to me. What's "OR" stand for?

You'll see that the array only has six elements, so while it shouldn't crash, sensor[6] does not exist (the array goes from 0 to 5). Per the [iRobot Coding protocol docs](https://github.com/iRobotEducation/root-robot-ble-protocol#device-11---ir-proximity), the seventh sensor is not exposed in iRobot Coding protocol version 1.4, but it will be exposed in 1.5.

**Reply by shamlian** [reply]
The seventh sensor should now be available in [release G.3.1](https://iroboteducation.github.io/create3_docs/releases/g_3_1/) using the Python function `async def get_packed_ir_proximity(self)` . Please let us know if you have any further trouble, but for now I am closing this ticket.
